apps/web/src/pages/Skills/components/SkillFilterDialog.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { OperationContext, useQuery } from "urql";
+
+import { Combobox, enumToOptions } from "@gc-digital-talent/forms";
+import { getLocalizedName } from "@gc-digital-talent/i18n";
+import { unpackMaybes } from "@gc-digital-talent/helpers";
+import {
+  SkillCategory,
+  SkillFamily,
+  graphql,
+} from "@gc-digital-talent/graphql";
+
+import adminMessages from "~/messages/adminMessages";
+import FilterDialog, {
+  CommonFilterDialogProps,
+} from "~/components/FilterDialog/FilterDialog";
+
+export type FormValues = {
+  skillFamilies?: string[];
+  skillCategories?: SkillCategory[];
+};
+
+const SkillFilterDialogData_Query = graphql(/* GraphQL */ `
```

Comment:
I honestly don't know the answer to this.  With the new codegen preset is it possible to send this as a fragment at the same time as SkillTableSkills instead of two different requests?

Reply:
Love it, fixed by [d9a1d63](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9614/commits/d9a1d633a370b466823792a8e697635996cf9655)